### ISSUE TITLE
Wrong menu icon color in Android.

### ISSUE THREAD

**Issue by Gloix** [GeekyAnts/NativeBase#1537]
## react-native, react and native-base version
react-native (0.52.0), react (16.2.0) and native-base (2.3.7)

## Expected behaviour
White menu icon on Android.

## Actual behaviour
The menu icon has the iOS blue color. (`iosToolbarBtnColor` | "#007aff")

## Steps to reproduce (code snippet or screenshot)
```
    <Header>
        <Left>
            <Button
                transparent
                onPress={() => this.props.navigation.navigate("DrawerOpen")}>
                <Icon name="menu" />
            </Button>
        </Left>
        <Body>
            <Title>HomeScreen</Title>
        </Body>
        <Right />
    </Header>
```

## Screenshot of emulator/device
![image](https://user-images.githubusercontent.com/5000394/35282346-03bd760a-0034-11e8-8df3-c6e6cf76c8c1.png)

## Any other additional info which would help us debug the issue quicker.
Not using CRNA, not declaring a custom theme. Already ran `react-native link`. Running in a physical device.

**Comment by SupriyaKalghatgi** [comment]
@Gloix Please verify the variables from [here](https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/variables/commonColor.js#L132) against your them

**Comment by Gloix** [comment]
Actually, version 2.3.7, which I have installed, has that line right
```
toolbarBtnColor: platform === "ios" ? "#007aff" : "#fff",
```
I also checked Platform.OS and it is Android.
I noticed I hadn't included the main theme, so I loaded it in, but the problem persists.

**Comment by akhil-geekyants** [comment]
@Gloix Check line [here](https://github.com/GeekyAnts/NativeBase/blob/v2.3.7/src/theme/variables/platform.js#L133) in version 2.3.7 .

**Comment by Gloix** [comment]
So, if I understand it right:
- `commonColor.js` declares common colors to all default themes
- `material.js` declares a material design like theme (although some checks are made for minor accomodations to the current platform?).
- `platform.js` declares a platform dependent theme.

I noticed the extracted theme from the theme setup instructions points to `variables/platform`, so it's using platform.js, and therefore is using the `toolbarBtnColor`property as @akhil-geekyants points:
```
toolbarBtnColor: "#007aff",
```
which actually produces the light blue menu tinting.

Now, is this color intended? I'm asking since every demo screen shown has a white menu icon like in the Readme file (and also looks more natural IMO):

![Android demo animation](https://github.com/GeekyAnts/NativeBase-KitchenSink/raw/master/screenshots/Android.gif)

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.8](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.8)
